test/babelish/test_csv2android.rb

```diff
@@ -35,4 +35,19 @@ def test_converting_csv_to_dotstrings_one_output_option
     # clean up
     system("rm -rf ./" + single_file)
   end
+
+  def test_converting_with_comments
+    csv_file = "test/data/test_data_with_comments.csv"
+    french_file = "values-fr/strings.xml"
+    expected_output = File.read("test/data/test_data_fr_with_comments.xml")
+    converter = Babelish::CSV2Android.new(csv_file,
+    { "French" => "fr" },
```

Comment:
@borut-t  https://github.com/bbatsov/ruby-style-guide#no-double-indent

here `{ "French" => "fr" }` should be align with csv_file